src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/XFIssue/AddingMultipleItemsListView.cs

```diff
@@ -13,43 +13,37 @@ public AddingMultipleItemsListView(TestDevice testDevice) : base(testDevice)
 
     public override string Issue => "Adding Multiple Items to a ListView";
 
-    // [Test]
-    // [FailsOnIOSWhenRunningOnXamarinUITest]
-    // public void AddingMultipleListViewTests1AllElementsPresent()
-    // {
-    //     App.WaitForElement("Big Job");
-    //     App.WaitForElement("Smaller Job");
-    //     App.WaitForElement("Add On Job");
-    //     App.WaitForElement("Add One");
-    //     App.WaitForElement("Add Two");
-    //     App.WaitForElement("3672");
-    //     App.WaitForElement("6289");
-    //     App.WaitForElement("3672-41");
-    //     App.WaitForElement("2");
-    //     App.WaitForElement("2");
-    //     App.WaitForElement("23");
-    //     App.Screenshot("All elements are present");
-    // }
-
-    // [Test]
-    // [FailsOnIOSWhenRunningOnXamarinUITest]
-    // public void AddingMultipleListViewTests2AddOneElementToList()
-    // {
-    //     App.Tap("Add One");
-
-    //     App.WaitForElement("1234", timeout: TimeSpan.FromSeconds(2));
-    //     App.Screenshot("One more element exists");
-    // }
-
-    // [Test]
-    // [FailsOnIOSWhenRunningOnXamarinUITest]
-    // public void AddingMultipleListViewTests3AddTwoElementToList()
-    // {
-    //     App.Screenshot("Click 'Add Two'");
-    //     App.Tap("Add Two");
-
-    //     App.WaitForElement("9999", timeout: TimeSpan.FromSeconds(2));
-    //     App.WaitForElement("8888", timeout: TimeSpan.FromSeconds(2));
-    //     App.Screenshot("Two more element exist");
-    // }
+	[Test]
+	public void AddingMultipleListViewTests1AllElementsPresent()
+	{
+		App.WaitForElement("Big Job");
+		App.FindElement("Big Job");
+		App.FindElement("Smaller Job");
+		App.FindElement("Add On Job");
+		App.FindElement("Add One");
+		App.FindElement("Add Two");
+		App.FindElement("3672");
+		App.FindElement("6289");
+		App.FindElement("3672-41");
+		App.FindElement("2");
+		App.FindElement("2");
+		App.FindElement("23");
+	}
+
+	[Test]
+	public void AddingMultipleListViewTests2AddOneElementToList()
+	{
+		App.Tap("Add One");
+
+		App.WaitForElement("1234", timeout: TimeSpan.FromSeconds(2));
+	}
+
+	[Test]
+	public void AddingMultipleListViewTests3AddTwoElementToList()
```

Comment:
Failing on Android:
<img width="1157" alt="image" src="https://github.com/user-attachments/assets/bd13d239-1afd-454f-80f2-cad3e4ce6578">

Reply:
And iOS.

Reply:
@jsuarezruiz  Resolved the failure and committed the changes, Please kindly review and share your concerns.

<img width="455" alt="image" src="https://github.com/user-attachments/assets/bc8dcd92-ede1-4222-bb31-f8c9561a1b0c">